src/client/search/searchActions.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+import { createAsyncActionType } from '../helpers/stateHelpers';
+import { getLookupAccountNames, getAllSearchResultPages } from '../helpers/apiHelpers';
+
+export const SEARCH_ASK_STEEM = createAsyncActionType('@search/SEARCH_ASK_STEEM');
+export const AUTO_COMPLETE_SEARCH = createAsyncActionType('@search/AUTO_COMPLETE_SEARCH');
+
+export const searchAskSteem = search => dispatch =>
+  dispatch({
+    type: SEARCH_ASK_STEEM.ACTION,
+    payload: {
+      promise: Promise.all([
+        getAllSearchResultPages(search)
+          .then((response) => {
+            let mergedResults = [];
+            _.each(response, (element) => {
+              mergedResults = _.concat(mergedResults, element.results);
+            });
+
+            return _.reverse(_.sortBy(mergedResults, ['type', 'created']));
+          })
+          .catch(() => dispatch({ type: SEARCH_ASK_STEEM.ERROR })),
```

Comment:
Wouldn't it dispatch anyway if we will remove `catch`?

Reply:
I'm not sure tbh, will it?